### PR TITLE
Get and view post comments

### PR DESCRIPTION
**Pull request by gracelungu**
#### What does this PR do?
This PR adds the functionality for getting and viewing post comments

#### Description of Task to be completed?
~ Add the `PostComments` component
~ Fetch the post comments within the actions
~ Test the component with its actions

#### How should this be manually tested?
~ Fetch and checkout to `ft-view-comments`
~ Add the `PostComments` components to `Home`
~ Pass the required `Post` props

#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Comment by gracelungu** [comment]
[Users should be able to view comments of a single post](https://trello.com/c/yrRExWHU/57-users-should-be-able-to-view-comments-of-a-single-post)

**Review by Karlmusingo — APPROVED** [review]

File: src/__tests__/actions/feed.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import server from '../../Api/server';
+import { getFeed } from '../../actions/feedActions';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import feedMocks from '../../__mocks__/feedMocks';
+
+const mockStore = configureMockStore([thunk]);
+
+describe('Login actions', () => {
```
Comment (nkpremices) [inline src/__tests__/actions/feed.test.js:9]: @gracelungu, is this test about login actions ?

File: src/__tests__/actions/postComments.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import server from '../../Api/server';
+import { getPostComments } from '../../actions/postComments';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import postCommentMock from '../../__mocks__/postCommentMock';
+
+const mockStore = configureMockStore([thunk]);
+
+describe('Login actions', () => {
```
Comment (nkpremices) [inline src/__tests__/actions/postComments.test.js:9]: Same here @gracelungu

**Review by nkpremices — CHANGES_REQUESTED** [review]

**Comment by oesukam** [comment]
@gracelungu Fix conflicts

**Pull request merged**